Telegram bot

```diff
@@ -0,0 +1,256 @@
+        bot.send_message(cid, 'Type the name of event');
+        bot.register_next_step_handler(message, get_name); #goes to the series of input functions
+    elif message.text == "/modify": # going to input functions, but other ones
+        if events(message)==[]:
+            bot.send_message(cid, 'Future prepares nothing');
+        else:
+            out=['The list of upcoming events:']+events(message)
+            bot.send_message(cid, '\n'.join(str(el) for el in out))
+            bot.send_message(cid, 'Type the number of event to edit without comma');
+            bot.register_next_step_handler(message, get_number);  
+    elif message.text == "/list": #List of upcoming events
+        if events(message)==[]:
+            bot.send_message(cid, 'Future prepares nothing');
+        else:        
+            out=['The list of upcoming events:']+events(message)
+            bot.send_message(cid, '\n'.join(str(el) for el in out))
+    elif message.text == "/notifications": #Always say "yes"
+        keyboarda = types.InlineKeyboardMarkup();
+        key_yes2 = types.InlineKeyboardButton(text='Yes', callback_data='yes2'); 
+        keyboarda.add(key_yes2); 
+        bot.send_message(cid, notstat(Plans.get(uid=cid).notification)+'\nWanna change?', reply_markup=keyboarda);
+    elif b==1:
+        b=0
+        if message.text == 'Edit name': #Here it comes from the "change_" functions and keyboard
+            bot.send_message(cid, 'Type new name for event');
+            bot.register_next_step_handler(message, change_name);  
+        elif message.text == 'Edit date':
+            bot.send_message(cid, 'Type new date for event (DD.MM.YYYY)');
+            bot.register_next_step_handler(message, change_date); 
+        elif message.text == 'Edit time':
+            bot.send_message(cid, 'Type new time for event (HH:MM, 24-hours)');
+            bot.register_next_step_handler(message, change_time); 
+        elif message.text == 'Remove':
+            keyboardy = types.InlineKeyboardMarkup(); 
+            key_yes1 = types.InlineKeyboardButton(text='Yes', callback_data='yes1'); 
+            keyboardy.add(key_yes1); 
+            key_no1 = types.InlineKeyboardButton(text='No', callback_data='no1');
+            keyboardy.add(key_no1);
+            bot.send_message(cid, 'Removing '+ename+' on ' +edate + ' at ' + etime +'\nConfirm?',reply_markup=keyboardy);
+            bot.register_next_step_handler(message, get_number); 
+        else:
+            bot.send_message(cid, 'Use the precious keyboard!!');
+            b=1
+
+        
+    else:
+        bot.send_message(cid, 'Try /new, /modify, /list or /notifications')
+
+def timecheck(time):
+    try:
+        datetime.strptime(time, '%H:%M') #I check if the user has read the instruction
+        return True
+    except Exception:
+        return False
+
+def datecheck(date):
+    try:
+        datetime.strptime(date, '%d.%m.%Y')
+        return True
+    except Exception:
```

Comment:
Is it possible to narrow down the range of exceptions?
It's usually a bad practice to process all possible exceptions